.gitignore

```diff
@@ -29,4 +29,6 @@ vendor/
 fastlane/report.xml
 google-services.json
 output-metadata.json
-service.properties
\ No newline at end of file
+service.properties
+wallet/schemas/*/*.json
```

Comment:
It seems like this line keeps appearing in our `.gitignore` in various PR's - there is at least one test that uses these files.

Reply:
Looks like it's removed from .gitignore in the head of master. Let's leave until the next merge.

Reply:
That is fine with me.